apps/nestjs-api/src/email/lib/email/email.js

```diff
@@ -322,7 +322,18 @@ export const sendEmailToUserWithTpJobseekerProfileSigningUpToCon = ({
 //   })
 // }
 
-export const sendMenteeSignupCompleteEmail = ({ recipient, firstName }) => {
+export const sendMenteeSignupCompleteEmail = ({ recipient, firstName, rediLocation }) => {
+  if (rediLocation === 'CYBERSPACE') {
+    const html = convertTemplateToHtml(null, 'signup-complete-mentee-cyberspace').replace(
+      /\${firstName}/g,
+      firstName
+    )
+    return sendMjmlEmailFactory({
+      to: recipient,
+      subject: 'Sign-up complete!',
+      html,
+    })
+  }
```

Comment:
You can simplify this a lot. As far as I can see, the only thing that's different from this code and the below is which template file is being loaded. So something like this would work:
```
const templateFile = (() => {
  if (rediLocation === 'CYBERSPACE') return 'signup-complete-mentee-cyberspace'
  else return 'signup-complete-mentee'
})()
```
and then pass `templateFile` to `convertTemplateToHtml()`

Reply:
Indeed. Thank you!